install_homebrew.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+# A simple Homebrew installation wrapper, to hide the platform-specific stanzas:
+# macOS: https://brew.sh/
+# Linux: https://docs.brew.sh/Homebrew-on-Linux#install
+
+set -e
```

Comment:
No `set -u` to catch undefined variables? (I suppose, why not the usual `set -euo pipefail` stanza?)

Reply:
So, it seems `-o pipefail` is not supported on `sh`, only bash - I guess for maximum portability `sh` is fine?